src/constants.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @author Flo Dörr
+ * @date 2019-02-04
+ */
+
+export const VERSION = 1;
+
+export const JITCOIN_DIR = './.jitcoin';
+
+export const BLOCKCHAIN_DIR = './.jitcoin/blockchain';
+
+export const JITCOIN_FILE = 'blk$.jit';
+
+export const JITCOIN_FILE_ZEROS = 8;
+
+// max file size in bytes (300kb for testing)
+//export const MAX_FILE_SIZE = 300000;
+export const MAX_FILE_SIZE = 10000000;
+
+// uncommon string to delimit the blocks
+export const DELIMITER = 'ĴḯŤ';
+
+export interface BLOCK_HEADER {
```

Comment:
Wouldn't it be better to put the interfaces in a separate folder/file? I get what you're trying to achieve here but I think a separate file gives it distinction.

Reply:
But then, I'd go with naming them `BlockHeader`, `BlockBody`, etc. to not break with the convention.

Reply:
good idea! Consider it done.